packages/core/src/particle/ParticleGenerator.ts

```diff
@@ -764,4 +766,150 @@ export class ParticleGenerator {
     out.push(vertexBufferBinding);
     return index;
   }
+
+  /** @internal */
+  _calculateLocalBoundingBox(): void {
```

Comment:
why not private method?

Reply:
get bounds in particleRenderer needs this method for recalculation